common/error_util.hpp

```diff
@@ -38,12 +38,12 @@ inline void report_and_clear_errno([[maybe_unused]] const std::string& file, [[m
                                    [[maybe_unused]] const std::string& function, [[maybe_unused]] const std::string& msg = "") {
 #ifndef NDEBUG
     if (errno > 0) {
-        if (ILLIXR::ENABLE_VERBOSE_ERRORS) {
+        // if (ILLIXR::ENABLE_VERBOSE_ERRORS) {
```

Comment:
`ENABLE_VERBOSE_ERRORS` is still defined above. We want to keep this config variable. Please change it according to the new way of defining configs.